packages/proto-ble/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "proto-ble",
+  "version": "3.9.1",
```

Comment:
Should we start it at 1.0.0?

Reply:
I think it'll get autolinked to the versions of the others once I do the first publish. So shouldn't really matter

Reply:
Ok np